csharp/private/rules/wrapper.bzl

```diff
@@ -2,12 +2,27 @@
 A wrapper around `dotnet` for Bazel.
 """
 
-load("//csharp/private:actions/wrapper.bzl", "write_wrapper_main_cc")
```

Comment:
Having this in another file was distracting. There are some bigger changes that need to happen to this stuff soon. I don't plan to reuse stuff at the action level.